lua/telescope/pickers.lua

```diff
@@ -1085,7 +1128,7 @@ function pickers.on_close_prompt(prompt_bufnr)
   -- TODO: This is an attempt to clear all the memory stuff we may have left.
   -- vim.api.nvim_buf_detach(prompt_bufnr)
 
-  picker.close_windows(status)
+  picker.close_windows(picker, status)
```

Comment:
and here we can do `picker:close_windows(status)`

Reply:
This was also fixed :)